marklogic-data-hub/src/main/resources/hub-internal-config/triggers/ml-dh-entity-create.json

```diff
@@ -30,6 +30,14 @@
     {
       "role-name": "%%mlFlowOperatorRole%%",
       "capability": "read"
+    },
+    {
+      "role-name": "%%mlDataHubDeveloperRole%%",
```

Comment:
These should be hard-coded to the role name, we don't want to allow for any overrides.

Reply:
now hardcoded